dipy/viz/streamline.py

```diff
@@ -1,12 +1,18 @@
+import warnings
 from dipy.utils.optpkg import optional_package
 
 plt, have_plt, _ = optional_package("matplotlib.pyplot")
 fury, has_fury, _ = optional_package('fury', min_version="0.10.0")
 if has_fury:
     from dipy.viz import actor, window
 
+sagital_deprecation_warning_msg = (
+    "The view argument value `sagital` is deprecated and its support will be"
```

Comment:
can you address the spelling errors ? (on both, variable name and msg)

Reply:
See https://github.com/dipy/dipy/pull/3218#issuecomment-2110966562.